certificatetprtest/service_test.go

```diff
@@ -0,0 +1,15 @@
+package certificatetprtest
+
+import (
+	"testing"
+
+	"github.com/giantswarm/certificatetpr"
+)
+
+func Test_Fake_Service_SearcherInterface(t *testing.T) {
+	newService := NewService()
+	_, ok := interface{}(newService).(certificatetpr.Searcher)
+	if !ok {
+		t.Fatal("fake service does not implement Searcher interface")
+	}
+}
```

Comment:
This test makes sure the fake implementation implements the `Searcher` interface, which is the only interface being defined in the certificatetpr package.